templates/resources/config_entry.md

```diff
@@ -280,3 +280,15 @@ The following attributes are exported:
 * `namespace` - The namespace to create the config entry within.
 
 * `config_json` - A map of configuration values.
+
+
+## Import
+
+`consul_config_entry` can be imported using the syntax `<kind>/<name>` if the
+config entry is in the default partition and default namespace, or
+`<kind>/<name>/<partition>/<namespace>` for config entries in a non-default
+partition or namespace:
```

Comment:
What about providing `foo/bar/default/default` is it equivalent to `foo/bar`?

Reply:
Yes it Is equivalent. Looking at it now it seems to me like `<partition>/<namespace>/<kind>/<name>` might make more sense here.

Reply:
I agree, that would be more inline with consul annotations around namespace and partition.

Reply:
Done!